packages/bundle-source/src/index.js

```diff
@@ -250,7 +282,7 @@ async function bundleNestedEvaluateAndGetExports(
   let sourceMap;
   let source;
   if (moduleFormat === 'getExport') {
-    sourceMap = `//# sourceURL=${resolvedPath}\n`;
+    sourceMap = `//# sourceURL=${DEFAULT_FILE_PREFIX}/${entrypoint}\n`;
```

Comment:
I think the default file prefix should be `...`, but I see it’s `/bundled-source/...`. I presume you verified this works for debuggers?

Reply:
Yes, for VSCode at least.